easybuild/tools/toolchain/linalg.py

```diff
@@ -127,7 +127,9 @@ def _set_blas_variables(self):
             self.variables.nappend('LIBBLAS', self.LIB_EXTRA, position=20)
             self.variables.nappend('LIBBLAS_MT', self.LIB_EXTRA, position=20)
 
+        self.variables.join('BLAS_SHARED_LIBS', 'LIBBLAS')
         self.variables.join('BLAS_STATIC_LIBS', 'LIBBLAS')
+        self.variables.join('BLAS_MT_SHARED_LIBS', 'LIBBLAS_MT')
         self.variables.join('BLAS_MT_STATIC_LIBS', 'LIBBLAS_MT')
```

Comment:
What is happening here? This looks strange to me that shared and static libs have something in common via LIBBLAS?

Reply:
This is strange internal stuff. The entry for `LIBBLAS_MT` is basically just a list of library names, at this point.

The values of each variable are later "formatted" according to the mapping in `easybuild/tools/toolchain/constants.py`, so the actual value you get for `$BLAS_MT_SHARED_LIBS` and `$LIBBLAS_MT` is different.